repose-aggregator/src/docs/asciidoc/recipes/docker.adoc

```diff
@@ -76,24 +76,32 @@ Now if we wanted to, we could change the Docker options that define the environm
 If we wanted to forward random ports instead of explicitly declaring the port mapping, we could replace the `--publish 8080:8080` option with `--publish-all` instead.
 
 == Manually Building a Repose Docker Image
-The latest version of the Dockerfiles used to build the *Repose Docker* images published on Docker Hub can be found at:
+The Dockerfile's used to build v{project-version} of the *Repose Docker* images published on Docker Hub are:
 
-- https://raw.githubusercontent.com/rackerlabs/repose/master/repose-aggregator/artifacts/docker/src/docker/ubuntu/Dockerfile[Debian/Ubuntu]
-- https://raw.githubusercontent.com/rackerlabs/repose/master/repose-aggregator/artifacts/docker/src/docker/centos/Dockerfile[RedHat/Centos]
++++ <details><summary>Ubuntu</summary><div> +++
```

Comment:
Not safe or recommended. This line assumes that HTML will be the output format, which makes our documentation far less portable. 

To use HTML in our documentation, we should use [Conditional Preprocessor Directives](http://asciidoctor.org/docs/user-manual/#conditional-preprocessor-directives). It's pretty similar to what you have here, with one more step. Check out [the includable search document](https://raw.githubusercontent.com/rackerlabs/repose/master/repose-aggregator/src/docs/asciidoc/_includes/search.adoc) for an example.

Reply:
As a follow-up, that this is being used to make the included `Dockerfile`s hideable indicates that they will not display well as full includes. Did we actually want to switch from links to includes, or did we want to copy and serve the `Dockerfiles` from our site and just link to them here?

Copying files into our output isn't too hard, and you can see an example of it here:
https://github.com/rackerlabs/repose/blob/master/repose-aggregator/build.gradle#L23-L36

Basically, have Gradle perform a literal file copy in a `Copy` task, then make the `asciidoctor` task depend on the `Copy` task.

Reply:
Updated.